### ISSUE TITLE
Australia: add population data, clean up neighbourhood records

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#1495]
Add population data (and population rank, etc) to locality, localadmin, and macrocountry records in Australia that do not have population values. Also, if possible, verify existing popoulation values when present in records.

Also, as a follow up to #1483, clean up neighbourhood points in Australia that share names with adjacent locality polygons. In #1483, we traversed the hierarchy and checked name matches of neighbourhood/locality pairs, but there is follow-up work to do to de-dup neighbourhoods "next to" localities with the same name

**Comment by nvkelso** [comment]
The `macrocountry` based `locality` records (~100 of them) are of top importance here, so the right **Sydney** & etc can come up in search results.

**Comment by NickStallman** [comment]
Would it be beneficial if I could provide a CSV of WOF IDs and population? Based on 2016 Bureau of Statistics data which is likely the most accurate source.
The data is also granular down to the locality level so basically every level in the heirachy can have an accurate population.

**Comment by nvkelso** [comment]
@NickStallman Yes, that would be fantastic and easy to integrate! :)

**Comment by NickStallman** [comment]
Here are my results. The CSV columns are:
PSMA ID, PSMA Raw Name, Name Cleaned, WOF place type, WOF ID, WOF Name, Population
[abs_wof_population.zip](https://github.com/whosonfirst-data/whosonfirst-data/files/2927698/abs_wof_population.zip)

I'm getting a poor match rate so I've opened issue #1509 however the matches I do have should be high quality and suitable for directly integrating in with the WOF dataset.

The population figures are from the 2016 Census data (G01 Tot_P_P). If there is any other census information which is relevant to WOF I can dump that out also in a similar format.

**Comment by nvkelso** [comment]
Noting this as come up a few more times as important.

**Comment by NickStallman** [comment]
Was there any progress on this? Can the data from my files be added or is any additional data required?

**Comment by nvkelso** [comment]
If I recall correctly the next step is to investigate the poor match rate in https://github.com/whosonfirst-data/whosonfirst-data/issues/1495#issuecomment-469360086. @stepps00 had some ideas about that back in March. Can you summarize them in a comment here, please?
